src/main/java/seedu/address/model/person/employee/Shift.java

```diff
@@ -17,6 +18,7 @@
     public static final String VALIDATION_REGEX =
```

Comment:
hmm Is the `VALIDATION_REGEX` still needed?

Reply:
Worked so hard for that validation regex i feel sad removing it HAHA

Reply:
LOL